### PR TITLE
change required number of arguments and add comments

### PR DESCRIPTION
**Pull request by quangx**
Previously there was a flag to determine whether the sampler would automatically detect bounds or if the user would input them. Now we simply count the number of arguments.

### CODE REVIEW

**Comment by quangx** [comment]
I am not quite sure how to indent this properly. Also I have a separate branch for spherical, I was thinking to pull request it into a separate branch but I can pull request to main if that's more appropriate.

File: sample/sample.cc [hunk 1]
```diff
@@ -140,6 +125,18 @@ class MyReader: public DataOutReader<dim,dim>
              
             }
           }
+          if(vertex(0)==0 && vertex(1)==1 && vertex(2)==1){
```
Comment (tjhei) [inline sample/sample.cc]: this is debug output, right? It should probably not part of this PR.

File: sample/sample.cc [hunk 2]
```diff
@@ -194,15 +185,16 @@ const Point<3,double> &p1,const Point<3,double> &p2,const std::array<unsigned in
...
 
-  // std::cout << "OK" << std::endl;
+ 
 }
 
-// ./sample 1 infile outfile minx max_x miny maxy minz maxz nx ny nz
-// ./sample 0 infile outfile nx ny nz
+
+
+// ./sample infile outfile minx max_x miny maxy minz maxz nx ny nz
```
Comment (tjhei) [inline sample/sample.cc]: can you output this as a simple help text if the number of arguments is incorrect? You are also missing the 2 argument case.

**Comment by tjhei** [comment]
> I am not quite sure how to indent this properly

I will add a script.

**Review by tjhei — APPROVED** [review]
